README.md

```diff
@@ -167,10 +161,25 @@ We are a team of engineers who work on the Developer Experience team at [Braintr
 
 Short answer: to accept PayPal as a payment option when mobile apps are using a WebView to power the checkout process.
 
-PayPal used to support authentication via a modal iframe, but authentication now occurs in a popup window to increase user confidence that their account information is protected from malicious actors (the address bar shows `https://checkout.paypal.com` with the HTTPS lock icon). However, this causes issues with Braintree merchants who use a web page to power payments within their apps: they can't accept PayPal because WebViews cannot open popups and return the PayPal payment authorization data to the parent checkout page.
+PayPal used to support authentication via a modal iframe, but authentication now occurs in a popup window to increase user confidence that their account information is protected from malicious actors (the address bar shows `paypal.com` with the HTTPS lock icon). However, this causes issues with Braintree merchants who use a web page to power payments within their apps: they can't accept PayPal because WebViews cannot open popups and return the PayPal payment authorization data to the parent checkout page.
```

Comment:
Mixing can't and cannot is the same sentence sounds weird. Might be worth updating this to one or the other.